src/main/java/com/autotune/utils/KruizeConstants.java

```diff
@@ -697,8 +697,9 @@ public static final class KRUIZE_CONFIG_DEFAULT_VALUE {
 
     public static final class KRUIZE_RECOMMENDATION_METRICS {
         public static final String METRIC_NAME = "KruizeRecommendationsNotification";
-        public static final String TAG_NAME = "experiment_details";
-        public static final String notification_format = "%s|%s|%s|%s|%s|%s|%s|%s|%s"; //experiment_name,container_name,endtime,level,termname,modelname,code,type,message
+        public static final String TAG_NAME = "recommendations_notifications";
+        public static final String notification_format_for_KIBANA = "%s|%s|%s|%s|%s|%s|%s|%s|%s"; //experiment_name,container_name,endtime,level,termname,modelname,code,type,message
```

Comment:
Can we avoid "kibana" naming in " String notification_format_for_KIBANA" as well ?

Reply:
Done